src/components/05_pages/Content/Content.js

```diff
@@ -157,7 +157,6 @@ class Content extends Component {
               {[
                 'Title',
                 'Content Type',
-                'Author',
```

Comment:
I'll take care of adding the author in a followup. It shouldn't be too difficult.